From 3596abbd7bb34d159069511663962be18c191ee5 Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Tue, 21 Jul 2026 01:43:14 +0000
Subject: [PATCH 08249/15333] ASoC: ti: ams-delta: use &pdev->dev instead of
 card->dev

ams_delta_probe() will be updated when Card capsuling.
To makes its review easy, use &pdev->dev instead of card->dev.
These are same

	card->dev = &pdev->dev;

No functional change, but is preparation for Card capsuling.

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Link: https://patch.msgid.link/87h5lti0cu.wl-kuninori.morimoto.gx@renesas.com
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 sound/soc/ti/ams-delta.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/ti/ams-delta.c b/sound/soc/ti/ams-delta.c
index 61252359d5cb..6a0a3a2073bf 100644
--- a/sound/soc/ti/ams-delta.c
+++ b/sound/soc/ti/ams-delta.c
@@ -562,12 +562,12 @@ static int ams_delta_probe(struct platform_device *pdev)
 
 	card->dev = &pdev->dev;
 
-	handset_mute = devm_gpiod_get(card->dev, "handset_mute",
+	handset_mute = devm_gpiod_get(&pdev->dev, "handset_mute",
 				      GPIOD_OUT_HIGH);
 	if (IS_ERR(handset_mute))
 		return PTR_ERR(handset_mute);
 
-	handsfree_mute = devm_gpiod_get(card->dev, "handsfree_mute",
+	handsfree_mute = devm_gpiod_get(&pdev->dev, "handsfree_mute",
 					GPIOD_OUT_HIGH);
 	if (IS_ERR(handsfree_mute))
 		return PTR_ERR(handsfree_mute);
-- 
2.55.0

